src/OpenLoco/src/Widget.cpp

```diff
@@ -559,7 +556,7 @@ namespace OpenLoco::Ui
 
         int width = widget->right - widget->left - 2;
         auto& drawingCtx = Gfx::getDrawingEngine().getDrawingContext();
-        drawingCtx.drawStringLeftClipped(*rt, x, y, width, colour, string, _commonFormatArgs);
+        drawingCtx.drawStringLeftClipped(*rt, x, y, width, colour, string, &FormatArguments::common());
```

Comment:
i take it you plan to make a window function that will get the args for these in the future so we can cut out using common?

Reply:
That's the idea yeah. Was thinking of splitting it off from the prepareDraw event, or passing an args struct into that event to use.